### PR TITLE
Added pinv to tensorflow frontend.

### PR DESCRIPTION
**Pull request by JoelPendleton**
#3615

### CODE REVIEW

**Comment by JoelPendleton** [comment]
Would it be possible to get feedback on why my code did not pass the tests?

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py [hunk 1]
```diff
@@ -174,5 +174,35 @@ def test_tensorflow_slogdet(
         fw=fw,
         frontend="tensorflow",
         fn_tree="linalg.slogdet",
-        x=np.asarray(x, dtype=input_dtype),
+        x=np.asarray(x, dtype=input_dtype)
+    )
+
+# pinv
+@given(
```
Comment (Unevilicorn) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py:181]: lint errors in this function

File: ivy/functional/frontends/tensorflow/linalg.py [hunk 2]
```diff
@@ -27,3 +27,8 @@ def slogdet(input, name=None):
 
 
 slogdet.unsupported_dtypes = ("float16", "bfloat16")
+
+def pinv(a, rcond=None, validate_args=False, name=None):
```
Comment (Unevilicorn) [inline ivy/functional/frontends/tensorflow/linalg.py:31]: lint errors in this function

**Review by Unevilicorn — CHANGES_REQUESTED** [review]
The function looks good, the only ones not passing for this function are the lint tests.

**Comment by Unevilicorn** [comment]
Looks good, I'll get this merged

**Pull request merged**